src/python/pants/backend/native/tasks/native_external_library_fetch.py

```diff
@@ -125,9 +127,10 @@ def _conan_pex_path(self):
 
   @memoized_property
   def _conan_binary(self):
-    return Conan.scoped_instance(self).bootstrap(
-      self._conan_python_interpreter,
-      self._conan_pex_path)
+    # Note: For historical reasons, this file manipulates and executes pex commandlines directly,
+    # instead of calling pex.run() (ideally in a workunit, via PythonToolInstance.run()).
+    # TODO: Switch to calling conan in a workunit.
+    return self.context.products.get_data(ConanPrep.tool_instance_cls).pex
```

Comment:
It might be that we can convert the whole native backend to v2 and do this at the same time.